README.md

```diff
@@ -3,7 +3,7 @@
 ## Get it running
 
 * Install [Node][1], NPM and build tools: `apt-get install nodejs nodejs-legacy build-essential`
-* Bump Node to version `4.2.x`: `sudo npm install n && sudo n ^4.2.0`
+* Bump Node to version `4.2.x`: `sudo npm install -g n && sudo n 4.2.3`
```

Comment:
Ah. Thanks for pointing this out. I believe you *can* bump it to `4.2.x` by doing `n 4.2`. Maybe it'd be better to change it to that so people copy-pasting will get the latest bug-fixes.